docker-compose.yml

```diff
@@ -43,8 +44,6 @@ services:
     restart: unless-stopped
     ports:
       - "127.0.0.1:5432:5432"
-    volumes:
```

Comment:
We should keep the volumes here, as folks tend to use them across container re-builds. If you need to delete PG data, just run `docker-compose down -v`

Reply:
Thank you! I reverted change and added info in CONTRIBUTING how to avoid malformed database between docker-compose ups.